index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="css/mobile-version.css"/>
+  <link rel="stylesheet" href="css/desktop-version.css"/>
+  <title>Youcode</title>
+</head>
+<body>
+  <!-- Main page section -->
+  <section id="main-page">
+    <header>
+      <nav id="navbar">
+        <a class="my-logo" href="#headline">YouCode</a>
+        <ul class="menu">
+          <li><a class="menuItem" href="#about-course">About</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Program</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">News</a></li>
+        </ul>
+        <ul class="desktop-menu">
+          <li><a class="" href="#about-course">About</a></li>
+          <li><a class="" href="#"style="color: lightgray">Program</a></li>
+          <li><a class="" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="" href="#" style="color: lightgray">News</a></li>
+         </ul>
+        <button class="hamburger">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+      </nav>
+    </header>
+    <main id="headline" class="headline">
+      <h1 class="title">A WEB DEVELOPMENT COURSE TO GET YOU STARTED</h1>
+      <p class="content">
+        Kickstart your courses in web development. Master html, css and javascript
+        courses using hands-on projects involving HTML, JavaScript, React.js, Ruby and more. 
+        No prior experience required.
+      </p>
+      <h2 class="date-set">2023.02.01(MON) - 15(THUR)</h2>
+      <p class="location">(Addis Ababa Arat Kilo Campus IT hall)</p>
+    </main>
+    <section class="main-program-list">
```

Comment:
- [ ] Kindly make sure to add the background Image to this section and to the title too as shown in the Behance design below:
- [ ] I think it would be better to add a small title about your main title as shown in the design below, but putting it between double quotes
Your design:
![Screenshot from 2023-02-10 23-08-12](https://user-images.githubusercontent.com/97834160/218197762-602d5a37-a9d9-40bc-82bf-0197a43cd36d.png)

Behance:

![Screenshot from 2023-02-10 23-08-44](https://user-images.githubusercontent.com/97834160/218197825-e65c5ae5-beb6-4db9-ada9-786cabe6c1e0.png)